contracts/container/contract.go

```diff
@@ -1105,38 +1105,38 @@ func PutReport(cid interop.Hash256, sizeBytes, objsNumber int, pubKey interop.Pu
 			storage.Put(ctx, append(reportersKey, counterToBytes(index)...), std.Serialize(reporter))
 		}
 	} else {
-		estInfo = ContainerEstimation{
-			SizeEstimation:          sizeBytes,
-			ObjectsNumberEstimation: objsNumber,
+		reportsSummary = NodeReportSummary{
+			ContainerSize:   sizeBytes,
+			NumberOfObjects: objsNumber,
 		}
 		reporter := NodeReport{PublicKey: pubKey, NumberOfReports: 1, ContainerSize: sizeBytes, NumberOfObjects: objsNumber}
 		storage.Put(ctx, append(reportersKey, counterToBytes(0)...), std.Serialize(reporter))
 	}
 
-	storage.Put(ctx, infoKey, std.Serialize(estInfo))
-	runtime.Log("saved container size estimation")
+	storage.Put(ctx, summaryKey, std.Serialize(reportsSummary))
+	runtime.Log("saved container size report")
 }
 
-// GetEstimation method returns an average load container's storage
-// nodes have and reported with [PutReport].
+// GetNodeReportSummary method returns a sum of object count and occupied
+// volume as reported by storage nodes provided via [PutReport].
 //
 // If the container doesn't exist, it panics with [cst.NotFoundError]. If no
 // reports were claimed, it returns zero values.
-func GetEstimation(epoch int, cid interop.Hash256) ContainerEstimation {
+func GetNodeReportSummary(epoch int, cid interop.Hash256) NodeReportSummary {
```

Comment:
Not related to this line, but: a comment to GetContainerSize should be adjusted, it contains a reference to GetEstimation.